extension/src/dashboard/pages/Authentication/LoginForm.tsx

```diff
@@ -17,6 +17,7 @@ import useUser from 'src/hooks/use-user';
 
 // TODO: make it common ?
 import { useFormErrors, GlobalErrorHandler, FieldErrorsHandler } from 'src/popup/components/Form';
+import { classList } from 'src/utils/classList';
```

Comment:
Why use `classList` here instead of `clsx` as we already did?
Or why we used `clsx` instead of `classList`?
Should we uniformized and chose one over another?

Reply:
didn't remember about clsx, I used `cx`, but it was not found in the imports, so I thought we didn't have any class list tool in this project